work-tracker-servlet/src/main/java/com/deere/isg/worktracker/servlet/AbstractHttpWorkFilter.java

```diff
@@ -55,13 +53,21 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
         } catch (ClassCastException e) {
             throw new ServletException(e);
         } finally {
-            if (outstanding != null) {
-                logger.logEnd((HttpServletRequest) request, (HttpServletResponse) response, payload);
+            try {
+                if (outstanding != null) {
+                    logger.logEnd(httpRequest, (HttpServletResponse) response, payload);
+                }
+                postProcess(request, response, payload);
```

Comment:
Hmm.  Move this to just before the log.  We aren't responding to the user until postprocess is done, so it needs to be accounted for.